nautobot/extras/tables.py

```diff
@@ -915,6 +922,55 @@ class Meta(BaseTable.Meta):
         )
 
 
+class ObjectMetadataTable(BaseTable):
+    pk = ToggleColumn()
+    metadata_type = tables.Column(linkify=True)
+    assigned_object = tables.TemplateColumn(template_code=ASSIGNED_OBJECT, orderable=False)
+    actions = ButtonsColumn(
+        ObjectMetadata,
+        buttons=("delete"),
+    )
+    # This is needed so that render_value method below does not skip itself
+    # when metadata_type.data_type is TYPE_CONTACT_TEAM and we need it to display either contact or team
+    value = tables.Column(empty_values=[])
+
+    class Meta(BaseTable.Meta):
+        model = ObjectMetadata
+        fields = (
+            "pk",
+            "assigned_object",
+            "scoped_fields",
+            "value",
+            "metadata_type",
```

Comment:
Might be more intuitive as a default order?

```suggestion
            "metadata_type",
            "scoped_fields",
            "value",
```